src/main/java/com/liferay/upgrades/analyzer/project/dependency/detector/JSToolKitDetector.java

```diff
@@ -0,0 +1,60 @@
+package com.liferay.upgrades.analyzer.project.dependency.detector;
+
+import com.liferay.upgrades.analyzer.project.dependency.graph.builder.ProjectsDependencyGraphBuilder;
+import com.liferay.upgrades.analyzer.project.dependency.model.ProjectKey;
+
+import java.io.BufferedReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+
+public class JSToolKitDetector implements ProjectDetector {
```

Comment:
Rename class to JSPortletProjectDetector or something like that to reflect which kind of project this detector will detect